### ISSUE TITLE
taskcluster 1.0.0 and 1.0.1 don't include the async subdir

### ISSUE THREAD

**Issue by escapewindow** [taskcluster/taskcluster-client.py#58]
This breaks async usage of the client.
[This line](https://github.com/taskcluster/taskcluster-client.py/blob/master/setup.py#L79) needs to include `taskcluster.async` for `python>=3.5`.

And because `python<3.5` won't include `taskcluster.async`, that means releases will need to be packaged via `python>=3.5`, e.g.

```bash
python3.5 setup.py sdist
```

**Comment by jhford** [comment]
this will be fixed with #61 .  I'm not sure that we really need to force python3.5 to generate sdists, we can package the files just fine with python2.7, it's just copying files around.

**Comment by jhford** [comment]
This has been resolved with version 1.0.2 which I just released to Pypi

```
jhford-desktop:~ $ source /home/jhford/taskcluster/taskcluster-client.py/junk/bin/activate
(junk) jhford-desktop:~ $ pip install taskcluster==1.0.2
Collecting taskcluster==1.0.2
  Downloading taskcluster-1.0.2.tar.gz (90kB)
    100% |████████████████████████████████| 92kB 1.3MB/s 
Collecting requests<3,>=2.4.3 (from taskcluster==1.0.2)
  Using cached requests-2.13.0-py2.py3-none-any.whl
Collecting mohawk<0.4,>=0.3.4 (from taskcluster==1.0.2)
Collecting slugid<2,>=1.0.7 (from taskcluster==1.0.2)
Requirement already satisfied: six<2,>=1.10.0 in ./taskcluster/taskcluster-client.py/junk/lib/python3.5/site-packages (from taskcluster==1.0.2)
Collecting aiohttp (from taskcluster==1.0.2)
  Using cached aiohttp-1.2.0-cp35-cp35m-manylinux1_x86_64.whl
Collecting async_timeout (from taskcluster==1.0.2)
  Using cached async_timeout-1.1.0-py3-none-any.whl
Collecting chardet (from aiohttp->taskcluster==1.0.2)
Collecting multidict>=2.1.4 (from aiohttp->taskcluster==1.0.2)
  Using cached multidict-2.1.4-cp35-cp35m-manylinux1_x86_64.whl
Collecting yarl>=0.8.1 (from aiohttp->taskcluster==1.0.2)
  Using cached yarl-0.8.1-cp35-cp35m-manylinux1_x86_64.whl
Building wheels for collected packages: taskcluster
  Running setup.py bdist_wheel for taskcluster ... done
  Stored in directory: /home/jhford/.cache/pip/wheels/b1/17/58/e9f1f124e2771f49e6e5e8e403ab2983a09b1f6d956894dca1
Successfully built taskcluster
Installing collected packages: requests, mohawk, slugid, chardet, async-timeout, multidict, yarl, aiohttp, taskcluster
Successfully installed aiohttp-1.2.0 async-timeout-1.1.0 chardet-2.3.0 mohawk-0.3.4 multidict-2.1.4 requests-2.13.0 slugid-1.0.7 taskcluster-1.0.2 yarl-0.8.1
(junk) jhford-desktop:~ $ python
Python 3.5.2 (default, Sep 14 2016, 11:28:32) 
[GCC 6.2.1 20160901 (Red Hat 6.2.1-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import taskcluster.async
>>> a = taskcluster.async.Auth()
>>> a.ping()
<coroutine object Auth.ping at 0x7f1a88d624c0>
>>> 
```
